clusterman/cli/toggle.py

```diff
@@ -22,20 +23,37 @@
 from clusterman.args import add_scheduler_arg
 from clusterman.args import subparser
 from clusterman.aws.client import dynamodb
+from clusterman.aws.client import sts
 from clusterman.cli.util import timeout_wrapper
 from clusterman.util import AUTOSCALER_PAUSED
 from clusterman.util import autoscaling_is_paused
 from clusterman.util import CLUSTERMAN_STATE_TABLE
 from clusterman.util import parse_time_string
 
+logger = colorlog.getLogger(__name__)
+
+
+@timeout_wrapper
+def check_account_id(cluster) -> None:
+    current_account_id = sts.get_caller_identity()
```

Comment:
i think response type for this endpoint isn't actually account number string, see https://docs.aws.amazon.com/STS/latest/APIReference/API_GetCallerIdentity.html

try running the command with your changes and verify if it works

Reply:
You're correct, I fixed it. Thanks :)